### PR TITLE
feat(modeler): support modeler API

### PR DESCRIPTION
**Pull request by daddykotex**
This is different than other features because as I've explained to @jtheoof. Sandbox and production APIs are the same from a SDK user's perspective. This changes with modeler.

The sandbox modeler API allows for more operations and the production modeler API.

cc @jtheoof @blemoine

### CODE REVIEW

**Comment by dmasse76** [comment]
This will worth having an update of the readme.md.  So much interfaces have been added.

**Comment by daddykotex** [comment]
@dmasse76 I agree

File: src/main/java/com/mnubo/java/sdk/client/spi/ModelSDK.java [hunk 1]
```diff
@@ -1,18 +1,273 @@
...
+ *
+ * The target environment is determined by the configuration you use:
+ *  - sandbox: sandbox url, consumer key and consumer secret
+ *  - production: production url, consumer key and consumer secret
+ *
+ *  @see <a href="https://smartobjects.mnubo.com/documentation/api_modeler.html">
+ *      API Modeler online documentation
+ *  </a>
  */
 public interface ModelSDK {
```
[the remark this answers is not in the document]
Comment (jtheoof) [reply src/main/java/com/mnubo/java/sdk/client/spi/ModelSDK.java]: Thanks I'll have a look

**Comment by jtheoof** [comment]
Good for merge @dmasse76 @mvonthron ?

**Comment by daddykotex** [comment]
I'll update the documentation of all the SDKs in a second MR.

**Pull request merged**